### ISSUE TITLE
USB fails to initialize on an 8GB Pi4

### ISSUE THREAD

**Issue by mtoman** [balena-os/balena-raspberrypi#493]
The following error gets logged on an 8GB Pi4
```
[    6.035811] xhci_hcd 0000:01:00.0: xHCI Host Controller
[    6.035854] xhci_hcd 0000:01:00.0: new USB bus registered, assigned bus number 1
[    6.535910] xhci_hcd 0000:01:00.0: can't setup: -110
[    6.535939] xhci_hcd 0000:01:00.0: USB bus 1 deregistered
[    6.535980] xhci_hcd 0000:01:00.0: init 0000:01:00.0 fail, -110
[    6.536024] xhci_hcd: probe of 0000:01:00.0 failed with error -110
```
and as a result the USB ports on the Pi do not work and the bus is not recognized:
```
root@pi8g # lsusb -t
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=dwc_otg/1p, 480M
```
It only happens on the 8GB version, the 4GB version gets initialized properly:
```
root@pi4g # lsusb -t 
/:  Bus 03.Port 1: Dev 1, Class=root_hub, Driver=dwc_otg/1p, 480M
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/4p, 5000M
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/1p, 480M
    |__ Port 1: Dev 2, If 0, Class=, Driver=hub/4p, 480M
```

**Comment by t3chguy** [comment]
Related https://github.com/raspberrypi/firmware/issues/1402#issuecomment-636443295

**Comment by PatrickLambert1101** [comment]
Im having the same issue comparing 4GB and 8GB the usb mouse is not working for the 8gb, is this related to the Balena image? I have the 8GB working with raspbian

**Comment by jtonello** [comment]
Can we confirm this issue is with the RPi4 8GB and _not_ the RPi4 4GB?

**Comment by t3chguy** [comment]
> It only happens on the 8GB version, the 4GB version gets initialized properly:

**Comment by acostach** [comment]
Fix for this is present in the kernel 4.19.118 from 2.53.12+rev1 in commit

> c74b1b532540 usb: xhci: Raspberry Pi FW loader for VIA VL805

Can be verified only after https://github.com/balena-os/balena-raspberrypi/issues/524 is fixed.

**Comment by floion** [comment]
This is fixed in release `2.56.0+rev1`

**Comment by heacens** [comment]
Ubuntu server 20.04.4 LTS (GNU/Linux 5.4.0-1060-raspi aarch64)， also have this problem， 8g device can not detect any usb。
